ArduPlane/Log.cpp

```diff
@@ -216,6 +247,42 @@ void Plane::Log_Write_RC(void)
     Log_Write_AETR();
 }
 
+void Plane::Log_Write_Guided(void)
+{
+#if OFFBOARD_GUIDED == ENABLED
+    if (control_mode != &mode_guided) {
+        return;
+    }
+
+    if (guided_state.target_heading_time_ms != 0) {
+        logger.Write_PID(LOG_PIDG_MSG, g2.guidedHeading.get_pid_info());
+    }
+
+    if (guided_state.target_alt > 0.0f || (guided_state.target_airspeed_cm > 0.001f )) {
```

Comment:
is_positive() is better than magic thresholds in the code

Reply:
yep, will fix.